pcsx2/gui/Panels/PluginSelectorPanel.cpp

```diff
@@ -92,56 +92,56 @@ class PluginEnumerator
 	//   BadStream				- thrown if the provided file is simply not a loadable DLL.
 	//   NotEnumerablePlugin	- thrown if the DLL is not a PCSX2 plugin, or if it's of an unsupported version.
 	//
-	PluginEnumerator( const wxString& plugpath )
-		: m_plugpath( plugpath )
+	PluginEnumerator(const wxString& plugpath)
+		: m_plugpath(plugpath)
```

Comment:
```suggestion
	PluginEnumerator(wxString plugpath)
		: m_plugpath(std::move(plugpath))
```

It's a good practice to let constructors take ownership of arguments.